README.md

```diff
@@ -175,7 +181,8 @@ $ dapr run --app-id nodeapp --app-port 3000 --grpc-port 50002 node app.js
 Example of launching Dapr within a specific Docker network:
 
 ```bash
-$ dapr run --app-id nodeapp --redis-host dapr_redis --placement-host dapr_placement node app.js
+$ dapr init --redis-host dapr_redis --network dapr-network
```

Comment:
What s dapr_redis here?

Reply:
Name of the redis host. This is same as the previous code except moved it from dapr run to dapr init